winconf.pri

```diff
@@ -3,8 +3,11 @@
 DEFINES += BOOST_ASIO_DISABLE_CONNECTEX
 DEFINES += BOOST_EXCEPTION_DISABLE
 
+DEFINES += TORRENT_USE_LIBCRYPTO
+# TODO: remove following defines as they are not used since libtorrent >= 1.2
```

Comment:
@Chocobo1:  IMO, you should probably add this comment to the relevant `cmake` files as well.

Reply:
added.